### ISSUE TITLE
The recommended algorithm for rating，Is it possible to evaluate using the ranking evaluation function?

### ISSUE THREAD

**Issue by Kiteflyingee** [guoguibing/librec#221]
#The official document said: Ranking recommended rating procedures using the rating method, you may get results, but it is not correct. Recommended procedure for rating If I configured rec.librec.ranking = true and then use the ranking function to evaluate the result would be correct?

**Comment by allenjack** [comment]
The result should be fine. But the performance may be not as good as the ranking methods.
By the way, could you rephrase your question in English? It would benefit the whole community.

**Comment by Kiteflyingee** [comment]
Sorry, I have rephrassed my question.Thank you!
